### PR TITLE
Update webserver template

### PR DESCRIPTION
**Pull request by norshtein**
Three changes:
- Seperate nsg of jumpbox and server, now jumpbox and vmss are using different nsg
- Rename `devbox` to `jumpbox`
- Rename some other resources

### CODE REVIEW

File: terraform/webserver/templates/network_security_group.tf [hunk 1]
```diff
@@ -1,33 +1,39 @@
-resource "azurerm_network_security_group" "kunlun_network_security_group" {
-  name                = "${var.env_name}-nsg"
+resource "azurerm_network_security_group" "kunlun_jumpbox_network_security_group" {
```
Comment (bingosummer) [inline terraform/webserver/templates/network_security_group.tf]: My opinion is to include nsg in jumpbox.tf and web_server.tf. We don't need a separated tf for nsg.

**Review by bingosummer — CHANGES_REQUESTED** [review]

File: terraform/webserver/templates/network_security_group.tf (hunk 1 above)
Comment (norshtein) [reply terraform/webserver/templates/network_security_group.tf]: It makes sense. I'll change the code.

File: pkg/apis/infra_producer_test.go [hunk 2]
```diff
@@ -29,30 +30,29 @@ var _ = Describe("InfraProducer", func() {
...
+			Azure:          azureConfig,
 			LatestTFOutput: "",
-
 		}
 		infraProducer = NewInfraProducer(globalConfig)
 	})
 
 	AfterEach(func() {
-		os.RemoveAll(stateDir)
+		//	os.RemoveAll(stateDir)
```
Comment (bingosummer) [inline pkg/apis/infra_producer_test.go]: Uncomment it.

File: pkg/apis/infra_producer_test.go [hunk 3]
```diff
@@ -29,30 +30,29 @@ var _ = Describe("InfraProducer", func() {
...
-			SubscriptionID:	"5ef1a6b6-cbfc-4e9f-bdae-1afb35682e7e",
-			TenantID:	"262f9b5b-376d-4c2e-ba1e-35372fb45d7f",
-			ClientID:	"d6d209e8-ec28-4b62-9ff6-826a3e09f6e1",
-			ClientSecret:	"e97b306c-2941-459c-97f3-2fba79c71b76",
+			Region:         "eastus2",
+			Environment:    "AzureCloud",
+			SubscriptionID: "c4528d9e-c99a-48bb-b12d-fde2176a43b8",
+			TenantID:       "72f988bf-86f1-41af-91ab-2d7cd011db47",
+			ClientID:       "43585f21-4171-41ec-a912-02689dc9dc27",
+			ClientSecret:   "64785ae3-5c0f-4d60-be12-393f4d5ff57d",
```
Comment (bingosummer) [inline pkg/apis/infra_producer_test.go]: Don't change the fake ids.

**Review by bingosummer — CHANGES_REQUESTED** [review]

**Comment by norshtein** [comment]
This PR is ready to merge, please have a review.

**Review by bingosummer — APPROVED** [review]

**Pull request merged**